task 6

```diff
@@ -0,0 +1,22 @@
+class Class
```

Comment:
You should never call your class `Class`, Ruby has basic class `Class`. So, you extend basic class in this way. Please, rename this class

Reply:
Done